modular_skyrat/modules/xenos_skyrat_redo/code/xeno_types/rouny.dm

```diff
@@ -0,0 +1,84 @@
+/// SKYRAT MODULE SKYRAT_XENO_REDO
+
+#define RUNNER_BLUR_EFFECT "runner_evasion"
+
+/mob/living/carbon/alien/humanoid/skyrat/runner
+	name = "alien runner"
+	desc = "A short alien with sleek red chitin, clearly abiding by the 'red ones go faster' theorem and almost always running on all fours."
+	caste = "runner"
+	maxHealth = 150
+	health = 150
+	icon_state = "alienrunner"
+	/// Holds the evade ability to be granted to the runner later
+	var/datum/action/cooldown/alien/skyrat/evade/evade_ability
+	melee_damage_lower = 20
+	melee_damage_upper = 25
+	next_evolution = /mob/living/carbon/alien/humanoid/skyrat/ravager
+	on_fire_pixel_y = 0
+
+/mob/living/carbon/alien/humanoid/skyrat/runner/Initialize(mapload)
+	. = ..()
+	AddComponent(/datum/component/tackler, stamina_cost = 0, base_knockdown = 2, range = 10, speed = 2, skill_mod = 7, min_distance = 0)
+	evade_ability = new /datum/action/cooldown/alien/skyrat/evade()
+	evade_ability.Grant(src)
+
+	add_movespeed_modifier(/datum/movespeed_modifier/alien_quick)
+
+/mob/living/carbon/alien/humanoid/skyrat/runner/Destroy()
+	QDEL_NULL(evade_ability)
+	return ..()
+
+/mob/living/carbon/alien/humanoid/skyrat/runner/create_internal_organs()
+	internal_organs += new /obj/item/organ/internal/alien/plasmavessel/small/tiny
+	..()
+
+/datum/action/cooldown/alien/skyrat/evade
+	name = "Evade"
+	desc = "Allows you to evade any projectile that would hit you for a few seconds."
+	button_icon_state = "evade"
+	plasma_cost = 50
+	cooldown_time = 60 SECONDS
+	/// If the evade ability is currently active or not
+	var/evade_active = FALSE
+	/// How long evasion should last
+	var/evasion_duration = 10 SECONDS
```

Comment:
ten seconds, with our baseline lowered movespeed, their higher movespeed+good damage, + our preference for ranged weapons over loading everyone up with combat-knives, makes this ability far to oppressive, namely because speed+pseudo-invulnerability every 50 (because the first 10 you're still in the mode) seconds is far to oppressive